telegram/_payment/stars/transactionpartner.py

```diff
@@ -352,10 +406,15 @@ def __init__(
         subscription_period: Optional[dtm.timedelta] = None,
         gift: Optional[Gift] = None,
         affiliate: Optional[AffiliateInfo] = None,
+        premium_subscription_duration: Optional[int] = None,
+        # temporarily optional to account for changed signature
+        transaction_type: Optional[str] = None,
         *,
         api_kwargs: Optional[JSONDict] = None,
     ) -> None:
         super().__init__(type=TransactionPartner.USER, api_kwargs=api_kwargs)
+        if transaction_type is None:
+            raise TypeError("`transaction_type` is a required argument since Bot API 9.0")
```

Comment:
```suggestion
        # # tags: deprecated NEXT.VERSION, bot api 9.0
        if transaction_type is None:
            raise TypeError("`transaction_type` is a required argument since Bot API 9.0")
```

just to find it more easily when doing the cleanup-PR :)
Could you also add a note in the chango fragment encouraging users to use keyword arguments since the signature will change in future release? I think putting this into the deprecations sections is okay

Reply:
I also left a place for `ChatFullInfo.accepted_gift_types` ( newly required, temporarily optional) in the change note.
will add a follow-up commit to the master PR after this gets merged.